packages/react-avatar/SPEC.md

```diff
@@ -35,25 +35,34 @@ Both components support showing a badge to indicate status, but they have differ
 
 ## Sample Code
 
-Basic examples:
+Display a user's initials:
 
 ```jsx
 <Avatar name="Miguel Garcia" />
-<Avatar size={72} name="Mona Kane" image="./MonaKane.jpg" />
-<Avatar shape="square" icon={<IDBadgeIcon />} />
 ```
 
-Displaying a badge (\*\*subject to change pending final spec for the `Badge` component):
+Display an image of a user:
```

Comment:
Nit: we should be consistent and reword this to "Display a user's image" or "Display user image"